src/types.ts

```diff
@@ -121,6 +122,11 @@ export interface CursorErrorEvent {
   error: CursorError
 }
 
+export interface ProcessErrorEvent {
+  type: 'processError'
+  error: unknown
```

Comment:
Do we want to also include the context of whether the error occurred during the initial scan vs. while processing the change stream?

Reply:
Good call!